test/models/financialDetails/ChargeItemSpec.scala

```diff
@@ -44,10 +45,14 @@ class ChargeItemSpec extends UnitSpec with ChargeConstants  {
     latePaymentInterestAmount = latePaymentInterestAmount,
     lpiWithDunningLock = lpiWithDunningLock)
 
-  val poa1FinancialDetails = financialDetail()
+  val docDetailsNoOutstandingAmout = defaultDocDetails.copy(outstandingAmount = 0)
 
+  val poa1FinancialDetails = financialDetail()
   val poa2FinancialDetails = financialDetail(mainTransaction = "4930")
 
+  val PoaOneReconciliationDebitDetails = financialDetail(mainTransaction = "4911")
```

Comment:
Please apply standard rules for Scala variable naming => not a bit issue though / we can find all these occurrences with use of ScalaFix in the future etc/ 
[](https://docs.scala-lang.org/style/naming-conventions.html)